html/donate.html

```diff
@@ -0,0 +1,494 @@
+    }
+
+    .submit-btn.disabled {
+        color: var(--bluegrey-light);
+        background-color: var(--grey-light);
+    }
+
+    #loader {
+        display: flex;
+        align-items: center;
+    }
+
+    #loader p {
+        font-size: 14px;
+        font-weight: 400;
+        line-height: 18px;
+    }
+
+    #instructions {
+        color: var(--disabled-text);
+    }
+
+    /* Utility classes */
+    .hidden {
+        display: none !important;
+    }
+
+    .disabled {
+        color: var(--disabled-text);
+    }
+
+    /* Spinned for loader */
+    .lds-ring {
+        display: inline-block;
+        position: relative;
+        width: 32px;
+        height: 32px;
+        margin-right: 16px;
+    }
+
+    .lds-ring div {
+        box-sizing: border-box;
+        display: block;
+        position: absolute;
+        width: 32px;
+        height: 32px;
+        margin: 2px;
+        border: 2px solid var(--main-blue);
+        border-radius: 50%;
+        animation: lds-ring 1.2s cubic-bezier(0.5, 0, 0.5, 1) infinite;
+        border-color: var(--main-blue) transparent transparent transparent;
+    }
+
+    .lds-ring div:nth-child(1) {
+        animation-delay: -0.45s;
+    }
+
+    .lds-ring div:nth-child(2) {
+        animation-delay: -0.3s;
+    }
+
+    .lds-ring div:nth-child(3) {
+        animation-delay: -0.15s;
+    }
+
+    @keyframes lds-ring {
+        0% {
+            transform: rotate(0deg);
+        }
+        100% {
+            transform: rotate(360deg);
+        }
+    }
+</style>
+<body onload="main()">
+<section>
+  <header>
+    <h1>A chi vuoi donare?</h1>
+    <a id="edit" class="hidden" href="#" onclick="resetSelection()">modifica</a>
+  </header>
+  <div id="entities-container"></div>
+  <div id="loader">
+    <div class="lds-ring">
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
+    <p>Stiamo caricando la lista delle organizzazioni</p>
+  </div>
+</section>
+
+<section class="how-much">
+  <header>
+    <h1 id="how-much-title" class="disabled">Quanto vuoi donare?</h1>
+  </header>
+  <ul class="tiers" id="tiers-container" onchange="selectTier(event)"></ul>
+  <p id="instructions">Scegli a chi donare per conoscere gli importi disponibili</p>
+</section>
+
+<section class="info-section">
+  <div id="info" class="info-box hidden">
+    <span class="info-icon">i</span>
+    <p class="info-text">
+      Puoi donare con carta di credito, debito o prepagata, senza costi di commissione.
+      La donazione è parzialmente detraibile
+    </p>
+  </div>
+</section>
+
+<section class="submit-section">
+  <button id="submit-btn" class="submit-btn disabled" onclick="submit()">Continua</button>
+</section>
+</body>
+
+<script>
+  let allCharities = "";
+
+  function fetchData() {
+    // Use this URL to get actual data
+    // const url = "https://api.dev.platform.pagopa.it/donations/availabledonations";
+    const url = "http://localhost:3000/donations/availabledonations";
```

Comment:
```suggestion
    const url = "https://api.dev.platform.pagopa.it/donations/availabledonations";
    //const url = "http://localhost:3000/donations/availabledonations";
```
For this asset we should do the opposite done in the dev server

Reply:
copy-paste shitshow at its best 😨 

Now the question: even if I hardcode the `api.dev...` url it will be the dev one. Shouldn't it be `https://api.dev.platform....` instead?

Reply:
I believe the better choice is to test the scenario at best we can (beta testing purposes). 
At the moment only dev is working

Change it when PROD is ready it's cheap and fast.
Does it make sense?

Reply:
here we go https://github.com/pagopa/io-services-metadata/pull/575/commits/35a00daa3b742cc94e9e44876077af28e9a947b2